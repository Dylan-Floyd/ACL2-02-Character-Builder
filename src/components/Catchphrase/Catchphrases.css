.catchphrases-container {
  width: 100%;
}

.catchphrases {
  height: 200px;
  overflow-y: auto;
  width: 100%;
  background-color: rgba(0, 0, 0, .15);
  border-radius: 10px;
}
