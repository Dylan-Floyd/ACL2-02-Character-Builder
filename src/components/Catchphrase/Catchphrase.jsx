import React from 'react'

import './Catchphrases.css'

export default function Catchphrase({
  catchphrases
}) {
  return (
    <div className='catchphrases-container'>
      Catchphrases:
      <div className='catchphrases'>
        {catchphrases.map(phrase => <p key={phrase}>{phrase}</p>)}
      </div>
    </div>
  )
}
