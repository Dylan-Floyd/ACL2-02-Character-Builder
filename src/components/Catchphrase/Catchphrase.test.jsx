import { screen, render } from '@testing-library/react'
import Catchphrase from './Catchphrase.jsx'

it('Should render the Catchphrase compononet', () => {
  const { container } = render(<Catchphrase catchphrases={ ['one', 'two'] } />)
  const catchphraseOne = screen.getByText('one')
  const catchphraseTwo = screen.getByText('two')

  expect(catchphraseOne).toBeInTheDocument()
  expect(catchphraseTwo).toBeInTheDocument()
  expect(container).toMatchSnapshot()
})
