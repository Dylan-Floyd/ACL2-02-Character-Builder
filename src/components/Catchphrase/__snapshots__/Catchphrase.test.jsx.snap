// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Catchphrase compononet 1`] = `
<div>
  <div>
    <div>
      Catchphrases:
      <p>
        one
      </p>
      <p>
        two
      </p>
    </div>
  </div>
</div>
`;
